Components/Decks/DraftDeckEditor.jsx

```diff
@@ -359,7 +360,7 @@ class DraftDeckEditor extends React.Component {
                     </div>
                 </div>
                 <div className='row'>
-                    <div className='col-sm-12' style={ { 'text-align': 'center', 'margin-bottom': '1em' } }>
+                    <div className='col-sm-12' style={ { 'textAlign': 'center', 'marginBottom': '1em' } }>
```

Comment:
I think the quotes are no longer needed if they're camel-cased.

Reply:
also fixed